app/models/episode.rb

```diff
@@ -365,7 +365,7 @@ def validate_media_ready
     end
   end
 
-  def transcript?
-    transcript.present?
+  def ready_transcript
```

Comment:
i feel like this just clicked a puzzle piece for me. i see this pattern throughout a lot of other models; access/return an attribute or association based on some conditions (otherwise returns nil?). like i feel like i see this a lot in the Flight model in augury, etc. and because this is such a simple example i think it makes a lot of sense.